### PR TITLE
Integrate notifications flyout.

### PR DESCRIPTION
**Pull request by reddyashish**
Integrate basic notifications fly-out. More changes will follow once we confirm its deployment on the Dynamo side. 

@QilongTang @filipeotero

### CODE REVIEW

**Comment by QilongTang** [comment]
@reddyashish It does not build with Node.js v16? what is your local version of it?

File: package-lock.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
       "version": "0.0.3",
       "license": "MIT",
       "dependencies": {
+        "@hig/notifications-flyout": "^3.2.0",
+        "@hig/theme-context": "^4.3.0",
+        "@hig/theme-data": "^3.0.0",
+        "@hig/timestamp": "^2.1.0",
```
Comment (QilongTang) [inline package-lock.json:15]: Should these be added as dev dependencies?

File: package.json [hunk 2]
```diff
@@ -18,7 +18,13 @@
     "build": "webpack --config webpack.config.js"
   },
   "keywords": [],
+  "author": "",
+  "license": "ISC",
   "dependencies": {
+    "@hig/notifications-flyout": "^3.2.0",
+    "@hig/theme-context": "^4.3.0",
+    "@hig/theme-data": "^3.0.0",
+    "@hig/timestamp": "^2.1.0",
```
Comment (QilongTang) [inline package.json:27]: Should we put them in the block below?

File: package.json [hunk 3]
```diff
@@ -18,7 +18,13 @@
     "build": "webpack --config webpack.config.js"
   },
   "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (QilongTang) [inline package.json:22]: These two lines have been moved by me to line 5 and 6. Please remove

File: src/App.js [hunk 4]
```diff
@@ -1,49 +1,48 @@
 import './App.css';
 import axios from 'axios';
 import React, { useEffect, useState } from 'react';
-import Moment from 'moment';
```
Comment (QilongTang) [inline src/App.js:4]: It seems we can remove the moment dev dependency

**Comment by QilongTang** [comment]
> > @reddyashish It does not build with Node.js v16? what is your local version of it?
> 
> It does, v16.13.2

Would you check out the Github Action build error?

**Comment by QilongTang** [comment]
When you have this cleaned up, would you also create a gif for the latest state?

File: src/App.js [hunk 5]
```diff
@@ -1,49 +1,48 @@
...
         setAPIData(response.data.notifications);
       })
   }, []);
-
+  
+  for (let i = 0; i < APIData.length; i++) 
+  {
+	var notificationItem = {
+							  id: APIData[i].id,
+							  featured: true,
```
Comment (QilongTang) [inline src/App.js:21]: What does featured mean?

File: src/App.js [hunk 6]
```diff
@@ -1,49 +1,48 @@
...
   const [APIData, setAPIData] = useState([]);
   useEffect(() => {
     axios.get(`http://demo9540080.mockable.io/notifications`)
       .then((response) => {
         setAPIData(response.data.notifications);
       })
   }, []);
-
+  
+  for (let i = 0; i < APIData.length; i++)
```
Comment (filipeotero) [inline src/App.js:17]: Did it stop from being looped more than once?

File: src/App.js [hunk 7]
```diff
@@ -1,49 +1,48 @@
...
+  for (let i = 0; i < APIData.length; i++) 
+  {
+	var notificationItem = {
+							  id: APIData[i].id,
+							  featured: true,
+							  unread: true,
+							  message: APIData[i].title,
+							  timestamp: <Timestamp timestamp= {APIData[i].created} />,
+							  content: <div>
+											<a style={{marginRight:'15px', marginTop:'-2px'}}><img width="20" src={APIData[i].thumbnail}/></a>
```
Comment (filipeotero) [inline src/App.js:26]: I remember I had some problems using 'px' values for our HTML pages in the guided tours. Maybe it's better to use 'rem' instead. https://nekocalc.com/px-to-rem-converter

File: src/App.js (hunk 6 above)
Comment (reddyashish) [reply src/App.js:17]: yes, as we are not using the map.

**Comment by reddyashish** [comment]
Created a clean PR to avoid the dependency conflicts. Closing this and will be using https://github.com/DynamoDS/NotificationCenter/pull/6

File: src/App.js (hunk 5 above)
Comment (reddyashish) [reply src/App.js:21]: A featured notification can be dismissed.

File: src/App.js (hunk 7 above)
Comment (reddyashish) [reply src/App.js:26]: Changed it in the new PR. More styling can be done once we confirm the workflow.

**Pull request closed without merging**